docroot/themes/contrib/civictheme/civictheme_library/components/03-organisms/alert/alert.scss

```diff
@@ -54,6 +54,10 @@
     @include ct-breakpoint(m) {
       position: relative;
     }
+
+    * {
```

Comment:
so, what if there are multiple paragraphs in the alert? message is a rich text field and may contain several paragraphs. 

please update this code to handle the last paragraph to not have a mergin